WalletWasabi/WabiSabi/Backend/WabiSabiConfig.cs

```diff
@@ -44,10 +44,10 @@ public WabiSabiConfig(string filePath) : base(filePath)
 	/// <summary>
 	/// The width of the rangeproofs are calculated from this, so don't choose stupid numbers.
 	/// </summary>
-	[DefaultValueMoneyBtc("1343.75")]
+	[DefaultValueMoneyBtc("43000")]
 	[JsonProperty(PropertyName = "MaxRegistrableAmount", DefaultValueHandling = DefaultValueHandling.Populate)]
 	[JsonConverter(typeof(MoneyBtcJsonConverter))]
-	public Money MaxRegistrableAmount { get; set; } = Money.Satoshis(ProtocolConstants.MaxAmountPerAlice);
+	public Money MaxRegistrableAmount { get; set; } = Money.Coins(43_000m);
```

Comment:
Why this change? You are also changing `ProtocolConstants.MaxAmountPerAlice` in the PR so why not keep using it?

Reply:
Because this is a **revert**, i mean, the result of doing `git revert <commit>` . If I get the ACK then i will continue with it, otherwise not.